sample/ApplicationInsightsHostingStartupSample/Startup.cs

```diff
@@ -20,13 +21,39 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
         }
 
-        public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory, IServiceProvider serviceProvider)
         {
-            loggerfactory.AddConsole(LogLevel.Debug);
+            loggerFactory.AddConsole(LogLevel.Debug);
```

Comment:
This is going to be a warning when I push https://github.com/aspnet/Logging/pull/605